build-support/flake8/.flake8

```diff
@@ -0,0 +1,22 @@
+[flake8]
+extend-ignore:
```

Comment:
There are so many new failures because `lint.pythonstyle` did not include Flake8 lints.